### PR TITLE
Move include outside of `postprocess_sim` function

### PR DESCRIPTION
**Pull request by imreddyTeja**
## Purpose 

#1120 moved post-processing for `run_amip.jl` into a function. Inside the function, there is an include for the leaderboard code. This moves the include outside of the function. 

## Content

----
- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

File: experiments/ClimaEarth/user_io/postprocessing.jl [hunk 1]
```diff
@@ -72,7 +73,6 @@ function postprocess_sim(::Type{AMIPMode}, cs, postprocessing_vars)
 
     # Check this because we only want monthly data for making plots
     if t_end > 84600 * 31 * 3 && output_default_diagnostics
-        include("leaderboard/leaderboard.jl")
```
Comment (charleskawczynski) [inline experiments/ClimaEarth/user_io/postprocessing.jl:75]: Can we also move `include("user_io/diagnostics_plots.jl")` into global scope? Julia does not permit loading code inside functions like that.

Comment (charleskawczynski) [reply experiments/ClimaEarth/user_io/postprocessing.jl:75]: And below (`include("user_io/diagnostics_plots.jl")`)

Comment (imreddyTeja) [reply experiments/ClimaEarth/user_io/postprocessing.jl:75]: Ah sorry, I didn't see all the includes inside the functions. Fixed now.

Comment (charleskawczynski) [reply experiments/ClimaEarth/user_io/postprocessing.jl:75]: It looks like the includes were not added, they were only removed

Comment (szy21) [reply experiments/ClimaEarth/user_io/postprocessing.jl:75]: It seems `diagnostics_plots.jl` is already included at the top of this file.

**Comment by szy21** [comment]
I'll get this in to see if it fixes the nightly amip.

**Review by szy21 — APPROVED** [review]

**Pull request merged**